worker/include/RTC/RTCP/FeedbackRtpTransport.hpp

```diff
@@ -290,7 +312,7 @@ namespace RTC
 
 		private:
 			uint16_t baseSequenceNumber{ 0u };
-			int32_t referenceTime{ 0 };
+			uint32_t referenceTime{ 0 };
```

Comment:
Are we sure of this? Reference time is supposed to be a 24bits signed value.

Reply:
Yes, the whole point of this commit is to not follow that rule, as libwebrtc doesn't care about sign for a reference_time already. https://webrtc.googlesource.com/src/+/refs/heads/main/modules/rtp_rtcp/source/rtcp_packet/transport_feedback.h#170